### PR TITLE
Js dscs 167

### PR DESCRIPTION
**Pull request by JonathanShaw**
refactor docworksSumbit to docworksSubmit, Fix comasters being used in re-ocr jobs

### CODE REVIEW

File: src/banjo/controllers/ReprocessController.java [hunk 1]
```diff
@@ -151,57 +151,67 @@ public String reDocworks(
...
+                List<Work> chapters = work.getPartsOf("chapter");
 
-            if (articles != null) {
-                for (Work article : articles) {
-                    log.debug("removing articel id {} for re-ocr work {}", article.getObjId(), objectId);
-                    Work w = amberSession.findWork(article.getId());
-                    amberSession.deleteWork(w);
+                if (articles != null) {
+                    for (Work article : articles) {
+                        log.debug("removing articel id {} for re-ocr work {}", article.getObjId(),
```
Comment (AaronCMuller) [inline src/banjo/controllers/ReprocessController.java]: typo?

File: src/banjo/services/ReprocessService.java [hunk 2]
```diff
@@ -299,7 +299,7 @@ private void createJobFileForJobItem(banjo.models.JobItem jobItem, CopyRole copy
 
                 file.setFileName(fileName);
                 JobFile jobFile = jobItem.createJobFile(fileName, fileFormat);
-                jobFile.modify().setAmberId(amberItem.getId()).save();
+                jobFile.modify().setCopyRole(copyRole.code()).setAmberId(amberItem.getId()).save();
```
Comment (AaronCMuller) [inline src/banjo/services/ReprocessService.java]: I presume this is the fix?

Comment (JonathanShaw) [inline src/banjo/services/ReprocessService.java]: yup

**Comment by AaronCMuller** [comment]
Merge away, once you fix that typo.

**Pull request merged**